test/string_utils_test.cc

```diff
@@ -28,5 +29,240 @@ TEST_F(StringUtilsTest, CheckJoin) {
+  EXPECT_EQ("123", to_string(123u));
+  EXPECT_EQ("4294967295",
+            to_string(std::numeric_limits<std::uint32_t>::max()));
+#if __x86_64__ || __ppc64__
+  EXPECT_EQ("0", to_string(0ll));
+  EXPECT_EQ("5", to_string(5ll));
+  EXPECT_EQ("123", to_string(123ll));
+  EXPECT_EQ("9223372036854775807",
+            to_string(std::numeric_limits<std::int64_t>::max()));
+  EXPECT_EQ("-9223372036854775808",
+            to_string(std::numeric_limits<std::int64_t>::min()));
+  EXPECT_EQ("0", to_string(0ull));
+  EXPECT_EQ("5", to_string(5ull));
+  EXPECT_EQ("123", to_string(123ull));
+  EXPECT_EQ("18446744073709551615",
+            to_string(std::numeric_limits<std::uint64_t>::max()));
+#endif  // __x86_64__ || __ppc64__
+  EXPECT_EQ("0.000000", to_string(0.0f));
+  EXPECT_EQ("1.000000", to_string(1.0f));
+  EXPECT_EQ("1.100000", to_string(1.1f));
+  EXPECT_EQ("1.234568", to_string(1.23456789f));
+  EXPECT_EQ("340282346638528859811704183484516925440.000000",
+            to_string(std::numeric_limits<float>::max()));
+  EXPECT_EQ("-1.000000", to_string(-1.0f));
+  EXPECT_EQ("-1.100000", to_string(-1.1f));
+  EXPECT_EQ("-12.345679", to_string(-12.3456789f));
+  EXPECT_EQ("-340282346638528859811704183484516925440.000000",
+            to_string(-std::numeric_limits<float>::max()));
+  EXPECT_EQ("0.000000", to_string(0.0));
+  EXPECT_EQ("1.000000", to_string(1.0));
+  EXPECT_EQ("1.100000", to_string(1.1));
+  EXPECT_EQ("1.234568", to_string(1.23456789));
+  EXPECT_EQ("17976931348623157081452742373170435679807056752584499659891747"
+            "68031572607800285387605895586327668781715404589535143824642343"
+            "21326889464182768467546703537516986049910576551282076245490090"
+            "38932894407586850845513394230458323690322294816580855933212334"
+            "8274797826204144723168738177180919299881250404026184124858368."
+            "000000",
+            to_string(std::numeric_limits<double>::max()));
+  EXPECT_EQ("-1.000000", to_string(-1.0));
+  EXPECT_EQ("-1.100000", to_string(-1.1));
+  EXPECT_EQ("-12.345679", to_string(-12.3456789));
+  EXPECT_EQ("-17976931348623157081452742373170435679807056752584499659891747"
+             "68031572607800285387605895586327668781715404589535143824642343"
+             "21326889464182768467546703537516986049910576551282076245490090"
+             "38932894407586850845513394230458323690322294816580855933212334"
+             "8274797826204144723168738177180919299881250404026184124858368."
+             "000000",
+            to_string(-std::numeric_limits<double>::max()));
+#if __x86_64__ || __ppc64__
+  /*
+   * We guarantee behavior of to_string(long double) only for x86_64 and ppc64
+   * operating systems.
+   */
+  EXPECT_EQ("0.000000", to_string(0.0L));
+  EXPECT_EQ("1.000000", to_string(1.0L));
+  EXPECT_EQ("1.100000", to_string(1.1L));
+  EXPECT_EQ("1.234568", to_string(1.23456789L));
+  EXPECT_EQ("118973149535723176502126385303097020516906332229462420044032373"
```

Comment:
This string literal is identical with that about the minimum value except the top `"-"` character, and both can be integrated into one string literal. Also `float` and `double` string literals can be integrated into one.